src/coreclr/jit/compiler.h

```diff
@@ -8960,6 +8960,39 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #endif
     }
 
+    //------------------------------------------------------------------------
+    // canUseEvexEncoding - Answer the question: Is Evex encoding supported on this target.
+    //
+    // Returns:
+    //    TRUE if Evex encoding is supported, FALSE if not.
```

Comment:
nit: for `bool` returning functions, write the comment using `true` and `false` that match the casing of the C++ keywords, and not `TRUE` and `FALSE`. For some of us, `TRUE` and `FALSE` map to the Win32 concepts with `BOOL` type. You can enclose them in single back-ticks like in GitHub markdown formatting if that helps distinguish them from other text, e.g.:
```
 //    `true` if Evex encoding is supported, `false` if not.
```

nit2: please include a blank `//` line at the end of the header comment block before the function so the comment text doesn't "run into" the function declaration.